source/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -322,7 +322,7 @@
     #error ipconfigRA_SEARCH_TIME_OUT_MSEC must be at least 0
 #endif
 
-STATIC_ASSERT( ipconfigRA_SEARCH_TIME_OUT_MSEC <= ( portMAX_DELAY * portTICK_PERIOD_MS ) );
+STATIC_ASSERT( pdMS_TO_TICKS( ipconfigRA_SEARCH_TIME_OUT_MSEC ) <= portMAX_DELAY );
```

Comment:
The maximum time can be now updated to `(portMAX_DELAY *  1000) / configTICK_RATE_HZ )`

Reply:
The problem is that `(portMAX_DELAY *  1000)` will overflow in many cases.
Appending `(portMAX_DELAY *  1000ull)` will not fix it for 64 bit architectures.

`( portMAX_DELAY / configTICK_RATE_HZ ) * 1000` does work but will have rounding errors.

It is just documentation, but users may copy it from there so what do you prefer?

Reply:
I believe the similar rounding errors can occur in the calculation of `pdMS_TO_TICKS( ipconfigRA_SEARCH_TIME_OUT_MSEC )` as well. This looks good to me.